packages/metal-soy/package.json

```diff
@@ -22,6 +22,7 @@
     "metal"
   ],
   "dependencies": {
+    "@webcomponents/custom-elements": "^1.0.0-alpha.3",
```

Comment:
Yes it probably should.

Note that I was using this dependency, because when launching test tests (on the transpiled code), I was getting this error:

```
 Uncaught TypeError: Failed to construct 'HTMLElement': Please use the 'new' operator, this DOM object constructor cannot be called as a function. 
```
There is more info about this [here](https://github.com/babel/babel/issues/4480) and [here](https://github.com/babel/babel/issues/51169)

Also, I might try to use [this](https://github.com/github/babel-plugin-transform-custom-element-classes) babel plugin which hopefully solves the issue.